CarboPlanner/src/views/HomeScreen.tsx

```diff
@@ -118,17 +118,21 @@ export default function HomeScreen(){
         </View>
       </Card>
 
-      <Card customStyle={{maxHeight: "44%"}}>
-      <FlatList
-        data={data}
-        renderItem={renderMealItem}
-        keyExtractor={(item, index) => index.toString()} // Use index as the key
-      />
-    </Card>
+      <ScrollView style={{paddingBottom: 10}}>
+        <Card customStyle={{maxHeight: "100%"}}>
+          <FlatList
+              data={data}
+              renderItem={renderMealItem}
+              keyExtractor={(item, index) => index.toString()} // Use index as the key
+          />
+        </Card>
+      </ScrollView>
       
       <View style={styles.cameraFunc}>
         <TouchableOpacity onPress={handlePress} style={styles.camera_button}>
-          <Text style={styles.photo}>+</Text>
+          <View style={styles.plusIconContainer}>
```

Comment:
Could you please extract this out into a sepereate component, as this would be the "correct" way to do it.